tools/automation_create_mentor_spreadsheets.py

```diff
@@ -0,0 +1,79 @@
+import pandas as pd
+import os
+import re
+import copy  # Import copy module for deep copying
+
+# Load the mentee registration data
+file_path = "Mentorship Programme long-term Registration Form for Mentees (Responses).xlsx"
+sheet_name = "Revised Mentees"
+df = pd.read_excel(file_path, sheet_name=sheet_name)  # Read the specified sheet from the Excel file
+
+# Define the relevant columns based on the example file
+columns_to_keep = [
+    "Mentee Id", "What is your full name?", "What is your email address?",
+    "Slack Name\nPlease note your application will be rejected if you are not in our Slack community.\nClick here to join us on Slack.",
+    "Where are you based? (Country and/or city)", "What is your current job title / education status?",
+    "Company / University name", "Your LinkedIn Profile", "How many years of experience do you have in the tech industry?",
+    "What tech skill you are most interested in? Mark your preference from 1 to 5 (1 - lowest, 5 - highest) [5]",
+    "What tech skill you are most interested in? Mark your preference from 1 to 5 (1 - lowest, 5 - highest) [4]",
+    "What tech skill you are most interested in? Mark your preference from 1 to 5 (1 - lowest, 5 - highest) [3]",
+    "What tech skill you are most interested in? Mark your preference from 1 to 5 (1 - lowest, 5 - highest) [2]",
+    "What tech skill you are most interested in? Mark your preference from 1 to 5 (1 - lowest, 5 - highest) [1]",
+    "What is your preferred programming language? Mark your preference from 1 to 5 (1 - lowest, 5 - highest) [5]",
+    "What is your preferred programming language? Mark your preference from 1 to 5 (1 - lowest, 5 - highest) [4]",
+    "What is your preferred programming language? Mark your preference from 1 to 5 (1 - lowest, 5 - highest) [3]",
+    "What is your preferred programming language? Mark your preference from 1 to 5 (1 - lowest, 5 - highest) [2]",
+    "What is your preferred programming language? Mark your preference from 1 to 5 (1 - lowest, 5 - highest) [1]",
+    "Please share your goals and expectations for this mentorship programme",
+    "Did you participate in the previous mentorship cycle in 2024?",
+    "Please describe how much experience you have in the area you would like to be mentored in. \n\nIf you are studying, tell us about your accomplished courses, projects, achievements, or interests",
+    "How many hours per week would you be able to dedicate to mentoring? (on average)",
+    "Why do you believe these mentor(s) can help you achieve your goals this year?\n\nPlease include which aspects of the mentor’s profile interest you the most and how they align with the skills the mentor offers and the ones you are also interested in developing."
+]
+
+# Ensure output directory exists
+output_dir = "Long Term Mentors"
```

Comment:
Could you extract this to path so we can change the path to another folder? but it can be done in another pr. For this I think It is really nice!! S2